frontend/src/pages/team/Billing.vue

```diff
@@ -116,8 +129,9 @@ export default {
                 })
                 this.loading = false
             } catch (err) {
-                // check for 404 and redirect if 404 returned
-                if (err.response.status === 404) {
+                // check for 402 and redirect if 402 returned
+                if (err.response.status === 402) {
+                    this.coupon = (await window.cookieStore.get('ff_coupon'))?.value.split('.')[0]
```

Comment:
`window.cookieStore` is not supported widely enough for us to use - https://developer.mozilla.org/en-US/docs/Web/API/CookieStore#browser_compatibility

Reply:
Pushed update without `window.cookieStore`